src/derivative_operators/abstract_operator_functions.jl

```diff
@@ -197,3 +233,32 @@ function *(A::AbstractDerivativeOperator,B::AbstractDerivativeOperator)
     #       the derivative of order A.dorder + B.dorder of
     #       approximation_order = min(approx_A, approx_B)
 end
+
+
+function convert(::Type{Array}, A::AbstractDerivativeOperator{T}, N::Int=A.dimension) where T
```

Comment:
why have two of this function? Please see the concretization.jl file. We don't need two.